cli/config/config.go

```diff
@@ -245,14 +309,56 @@ func (c *CommonOpts) GetFeatures() cnquery.Features {
 // GetServiceCredential returns the service credential that is defined in the config.
 // If no service credential is defined, it will return nil.
 func (c *CommonOpts) GetServiceCredential() *upstream.ServiceAccountCredentials {
-	if c.Authentication != nil && c.Authentication.Method == "ssh" {
-		log.Info().Msg("using ssh authentication method, generate temporary credentials")
-		serviceAccount, err := upstream.ExchangeSSHKey(c.UpstreamApiEndpoint(), c.ServiceAccountMrn, c.GetParentMrn())
-		if err != nil {
-			log.Error().Err(err).Msg("could not exchange ssh key")
-			return nil
+	// If we have an authentication method defined, use it
+	if c.Authentication != nil {
+		switch c.Authentication.Method {
+		case "ssh":
+			log.Info().Msg("using ssh authentication method, generate temporary credentials")
+			serviceAccount, err := upstream.ExchangeSSHKey(c.UpstreamApiEndpoint(), c.ServiceAccountMrn, c.GetParentMrn())
+			if err != nil {
+				log.Error().Err(err).Msg("could not exchange ssh key")
+				return nil
+			}
+			return serviceAccount
+		case "wif":
+			log.Info().Msg("using wif authentication method, generate temporary credentials")
+
+			// Validate required fields for WIF
+			if c.Audience == "" {
+				log.Error().Msg("missing required field 'audience' for WIF authentication")
+				return nil
+			}
+
+			// If apiEndpoint is not set, check for universeDomain in viper
+			if c.APIEndpoint == "" {
+				universeDomain := viper.GetString("universeDomain")
```

Comment:
How can we end up here? If we have `c.Authentication.Method == "wif"` I think it means we loaded the WIF config during `InitViperConfig` so this feels redundant and we should get rid of it unless we are covering some edge case that I am not seeing. 🤔

Reply:
Yeah this is definitely redundant, removed.